profiles/pci/handhelds/profiles.toml

```
@@ -160,4 +185,7 @@ post_remove = """
     for mask in ${maskservices[@]}; do
         systemctl unmask "${mask}.service"
     done
+    echo "Removing audio profile.."
+    rm /etc/pipewire/pipewire.conf.d/filter-chain.conf
```

Same as above, and for wireplumber config too.
```suggestion
    rm -f /etc/pipewire/pipewire.conf.d/filter-chain.conf
```